src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -228,6 +222,7 @@ private Slave prepareDockerPlatforms() throws Exception {
                 throw new AssertionError(platform);
         }
 
+        String arch = System.getProperty("os.arch");
```

Comment:
I think this is unused, right?

```suggestion
```

Reply:
you're right, removed in fcab5fac0547fd470de5427ad1b6832cc3154614